apps/fluent-tester/src/E2E/Badge/specs/Badge.spec.win.ts

```diff
@@ -22,3 +23,22 @@ describe('Badge Testing Initialization', function () {
     expect(BasicBadgePageObject.didAssertPopup()).toBeFalsy(BasicBadgePageObject.ERRORMESSAGE_ASSERT);
   });
 });
+
+describe('CounterBadge Testing', () => {
+  beforeEach(() => {
+    CounterBadgePageObject.scrollToTestElement();
+    CounterBadgePageObject.waitForPrimaryElementDisplayed(PAGE_TIMEOUT);
+  });
+  it('Validate display count', () => {
+    expect(CounterBadgePageObject.getPrimaryComponentText()).toEqual('77');
```

Comment:
I think this would be better as a Jest/Unit test and not an E2E test. Interactions shouldn't change the count, and since E2E tests are flakier than unit tests, my opinion is that we should use unit tests to verify props that don't require interactivity or some sort of e2e scenario.

@samuelfreiberg Curious on your thoughts?

Reply:
I think you're right. I also added it to unit tests

Reply:
I agree with Ruriko. It doesn't look like there's any UI manipulation happening here, so these tests are better suited for unit tests.

Reply:
Thank you :)